include/z64camera.h

```diff
@@ -27,6 +27,89 @@
 #define PARENT_CAM(cam) ((cam)->play->cameraPtrs[(cam)->parentCamId])
 #define CHILD_CAM(cam) ((cam)->play->cameraPtrs[(cam)->childCamId])
 
+#define CAM_HUD_VISIBILITY_SHIFT 8
+#define CAM_HUD_VISIBILITY_MASK (0x0F00)
+#define CAM_HUD_VISIBILITY(hudVisibility) (((hudVisibility) & 0xF) << CAM_HUD_VISIBILITY_SHIFT)
+
+#define CAM_HUD_VISIBILITY_50 CAM_HUD_VISIBILITY(0)
+#define CAM_HUD_VISIBILITY_1 CAM_HUD_VISIBILITY(1)
+#define CAM_HUD_VISIBILITY_2 CAM_HUD_VISIBILITY(2)
+#define CAM_HUD_VISIBILITY_3 CAM_HUD_VISIBILITY(3)
+#define CAM_HUD_VISIBILITY_4 CAM_HUD_VISIBILITY(4)
+#define CAM_HUD_VISIBILITY_5 CAM_HUD_VISIBILITY(5)
+#define CAM_HUD_VISIBILITY_6 CAM_HUD_VISIBILITY(6)
+#define CAM_HUD_VISIBILITY_7 CAM_HUD_VISIBILITY(7)
+#define CAM_HUD_VISIBILITY_8 CAM_HUD_VISIBILITY(8)
+#define CAM_HUD_VISIBILITY_9 CAM_HUD_VISIBILITY(9)
+#define CAM_HUD_VISIBILITY_A CAM_HUD_VISIBILITY(10)
+#define CAM_HUD_VISIBILITY_B CAM_HUD_VISIBILITY(11)
+#define CAM_HUD_VISIBILITY_C CAM_HUD_VISIBILITY(12)
+#define CAM_HUD_VISIBILITY_D CAM_HUD_VISIBILITY(13)
+#define CAM_HUD_VISIBILITY_IGNORE CAM_HUD_VISIBILITY(0xF)
+
+/**
+ * letterboxFlag: determines the size of the letter-box window. See CAM_LETTERBOX_* enums.
+ *                  Can also add on the flag ( | CAM_LETTERBOX_INSTANT) to make the window shrink immediately
+ * hudVisibility: hides certain hud icons
+ *    - A value of 0 in camera is translated to an hudVisibility of 50, 
```

Comment:
:smiling_imp: 
```suggestion
 *    - A value of 0 in camera is translated to a hudVisibility of 50, 
```

Reply:
Are you reading this as an acronym? I'd use "an" there since I don't.

Reply:
I'm probably plastering some rules from French onto English
probably mainly: https://en.wikipedia.org/wiki/Aspirated_h
but even then I would say like "a hero" not "an hero"

But it's really not a big deal and it can stay *an* if you want, I suggested *a* because I figured *an* was a left-over from the previous *an alphaType*

Reply:
https://en.wikipedia.org/wiki/English_alphabet#Letter_names
I pronounce it as "aitch-u-dee", that's why I expect "an".

Reply:
"an hud" doesnt sound right when saying it as a word. "a" sounds correct to me. even if saying the whole thing, "a heads up display" is correct

Reply:
I think its way more common to just say it as if it were its own word (similar to hood but with a u sound)

Reply:
Oh it is an acronym, weird. I blame the military-industrial complex.